src/test/java/multithreading/ExecutorServiceImplTest.java

```diff
@@ -0,0 +1,52 @@
+package multithreading;
+
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+
+public class ExecutorServiceImplTest {
+    private static final int SUM_POSITIVE_NUMBER = 10;
+    private static final int SUM_NEGATIVE_NUMBER = -10;
+    private static final int SUM_ZERO_NUMBER = 0;
+    private static final int WRONG_RESULT = 12;
+    private static List<Integer> listNumbers;
+
+    @BeforeAll
+    public static void beforeAll() {
+        listNumbers = new ArrayList<>();
+    }
+
+    @Test
+    public void checkCalculateSumOfPositiveValuesFromListOK() {
+        listNumbers = List.of(1, 2, 3, 4);
```

Comment:
Let's test on list with 1m elements. We don't need test with this input

Reply:
done